blocks/faq/src/app.tsx

```diff
@@ -0,0 +1,469 @@
+          {!readonly ? (
+            <Fade in={hovered}>
+              <Box
+                sx={{
+                  display: "flex",
+                  columnGap: 3,
+                  rowGap: 1,
+                  flexWrap: "wrap",
+                  mb: 1.5,
+                }}
+              >
+                <GetHelpLink href="https://blockprotocol.org/@hash/blocks/faq" />
+
+                <Box display="flex" flexWrap="wrap" rowGap={1} columnGap={3}>
+                  <Box display="flex" gap={1}>
+                    <Typography
+                      sx={{
+                        fontWeight: 500,
+                        fontSize: 15,
+                        lineHeight: 1,
+                        color: ({ palette }) => palette.gray[50],
+                      }}
+                    >
+                      Show numbers?
+                    </Typography>
+                    <Switch
+                      size="small"
+                      checked={displayNumbers}
+                      onChange={(event) => {
+                        setDisplayNumbers(event.target.checked);
+                        void updateField(
+                          event.target.checked,
+                          shouldDisplayQuestionNumbersKey,
+                        );
+                      }}
+                    />
+                  </Box>
+
+                  <Box display="flex" gap={1}>
+                    <Typography
+                      sx={{
+                        fontWeight: 500,
+                        fontSize: 15,
+                        lineHeight: 1,
+                        color: ({ palette }) => palette.gray[50],
+                      }}
+                    >
+                      Show toggles?
+                    </Typography>
+                    <Switch
+                      size="small"
+                      checked={displayToggles}
+                      onChange={(event) => {
+                        setDisplayToggles(event.target.checked);
+                        void updateField(
+                          event.target.checked,
+                          shouldDisplayQuestionTogglesKey,
+                        );
+                      }}
+                    />
+                  </Box>
+                </Box>
+              </Box>
+            </Fade>
+          ) : null}
+
+          <Card
+            sx={{
+              display: "flex",
+              flexDirection: "column",
+              gap: 3,
+              ...(!readonly
+                ? {
+                    border: ({ palette }) => `1px solid ${palette.gray[20]}`,
+                    borderRadius: 2.5,
+                    boxShadow: "none",
+                    paddingY: 3,
+                    paddingX: 3.75,
+                  }
+                : {
+                    background: "none",
+                    boxShadow: "none",
+                  }),
+            }}
+          >
+            {title || description || !readonly ? (
```

Comment:
Maybe this could be a variable with a more self-explanatory name like `shouldDisplayXYZ`

Reply:
Done in 8a5e5ad9cbd73d52e7dd235d08d964c40b2b2d39